src/fastga/models/geometry/geom_components/vt/components/compute_vt_chords.py

```diff
@@ -39,20 +39,63 @@ def setup(self):
         self.declare_partials(
             "data:geometry:vertical_tail:span",
             ["data:geometry:vertical_tail:aspect_ratio", "data:geometry:vertical_tail:area"],
-            method="fd",
+            method="exact",
         )
-        self.declare_partials("data:geometry:vertical_tail:root:chord", "*", method="fd")
-        self.declare_partials("data:geometry:vertical_tail:tip:chord", "*", method="fd")
+        self.declare_partials("data:geometry:vertical_tail:root:chord", "*", method="exact")
+        self.declare_partials("data:geometry:vertical_tail:tip:chord", "*", method="exact")
 
+    # pylint: disable=missing-function-docstring, unused-argument
+    # Overriding OpenMDAO compute, not all arguments are used
     def compute(self, inputs, outputs, discrete_inputs=None, discrete_outputs=None):
-        lambda_vt = float(inputs["data:geometry:vertical_tail:aspect_ratio"])
-        s_v = float(inputs["data:geometry:vertical_tail:area"])
-        taper_v = inputs["data:geometry:vertical_tail:taper_ratio"]
+        aspect_ratio = inputs["data:geometry:vertical_tail:aspect_ratio"]
```

Comment:
```suggestion
        aspect_ratio_vt = inputs["data:geometry:vertical_tail:aspect_ratio"]
```
Here and everywhere else in the component of course

Reply:
Variable naming corrected at c4ba6cd8.